masonite/drivers/BaseUploadDriver.py

```diff
@@ -0,0 +1,26 @@
+import os
+from masonite.exceptions import ExtensionException
+
+
+class BaseUploadDriver(object):
+    """
+    Class base for upload drivers
+    """
+    def __init__(self):
```

Comment:
This won't work either because drivers need to have their own constructor so a `BaseUploadDriver` cannot have it's own constructor. Try possibly doing something like:

```
class BaseUploadDriver(object):

    accept_file_types = None
```

instead of putting it in a constructor.

Let me know if that makes sense.